soh/src/overlays/actors/ovl_En_Box/z_en_box.c

```diff
@@ -717,6 +717,36 @@ void EnBox_UpdateSizeAndTexture(EnBox* this, PlayState* play) {
             this->boxLidDL = gChristmasGreenTreasureChestChestSideAndLidDL;
         }
     }
+
+    // Chest Sizes Match Contents can make certain chests unreachable, so nudge
+    // the ones that cause problems.
+    // https://github.com/gamestabled/OoT3D_Randomizer/blob/68cf3f190d319e554bdeebc7f16e67578430dbc3/code/src/actors/chest.c#L57
+    s16 params = this->dyna.actor.params;
+    s16 sceneNum = play->sceneNum;
+    s16 room = this->dyna.actor.room;
+    s16 isLarge = this->dyna.actor.scale.x == 0.01f;
+
+    // Make Ganon's Castle Zelda's Lullaby chest reachable when large.
+    if ((params & 0xF000) == 0x8000 && sceneNum == SCENE_GANONTIKA && room == 9) {
+        this->dyna.actor.world.pos.z = isLarge ? -962.0f : -952.0f;
```

Comment:
I'm not a huge fan of having to supply both the modified and the origin values here.

If you re-arrange the checks to be something like this, you can then just provide the modified values.
```
if (isLarge) {
  if (ganon) {
    ...
  }
  if (deku) {
    ...
  }
} else {
  ...
}
```

Reply:
I can't do that because the chests need to be able to return to their original positions when the enhancement is turned off. So I'd have to either keep it like it is, or add another position field to the chest actor that tracks its original position. If there were more chests that need to be moved, I'd consider doing that, but it feels like overkill when only four are affected.

Reply:
> I can't do that because the chests need to be able to return to their original positions when the enhancement is turned off.

Ah, yea I forgot about that aspect. Makes sense, and yea it's only 4 chests, no need to go over kill.